### PR TITLE
Basic services

### PR DESCRIPTION
**Pull request by JakobMiksch**
This adds basic services for the KLIPS SDI:
- RabbitMQ
- GeoServer
- A node script that initializes GeoServer
  - the docker image is currently built directly from the repo. But we can move the image to our GitHub docker registry as well
- the dispatcher service

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,15 @@ The spatial data infrastructure for the [KLIPS project](http://www.klips-projekt
 
 ## Setup
 
-
 ```shell
 # create your personal config
-cp .env.example .env
```
Comment (weskamm) [inline README.md:10]: i like the old syntax, no need to break things down, but its just personal opinion..

File: geoserver-init/010_create-layers.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import GeoServerRestClient from 'geoserver-node-client';
```
Comment (weskamm) [inline geoserver-init/010_create-layers.js:1]: nitpick: file is named `create-layers` but it just creates workspaces, or am i wrong?

**Review by weskamm — APPROVED** [review]
just 2 minors, no need to address them..

File: geoserver-init/010_create-layers.js (hunk 2 above)
Comment (JakobMiksch) [reply geoserver-init/010_create-layers.js]: good catch!

**Comment by JakobMiksch** [comment]
@weskamm I addressed the notes. Thanks for reviewing.

**Pull request merged**